.github/dependabot.yml

```diff
@@ -11,30 +11,18 @@ updates:
     schedule:
       interval: 'daily'
     commit-message:
-      prefix: '[DEPBOT-0101]'
-
-  # Enable version updates for Docker
-  - package-ecosystem: 'docker'
-    # Look for a `Dockerfile` in the `root` directory
-    directory: '/'
-    # Check for updates once a week
-    schedule:
-      interval: 'daily'
-    commit-message:
-      prefix: '[DEPBOT-0102]'
+      #Packaged dependencies (release)
+      prefix: fix
```

Comment:
update dependabot PR title prefix so when a PR is approved and merged, a new release is created.